### PR TITLE
fix - schema builder with schema in dbName

### PR DESCRIPTION
**Pull request by jycouet**

### CODE REVIEW

File: projects/core/src/column-interfaces.ts [hunk 1]
```diff
@@ -53,7 +53,13 @@ export interface FieldOptions<entityType = any, valueType = any> {
   ) => any | Promise<any>
   /**  An expression that will determine this fields value on the backend and be provided to the front end*/
   serverExpression?: (entity: entityType) => valueType | Promise<valueType>
-  /** The name of the column in the database that holds the data for this field. If no name is set, the key will be used instead. */
+  /** The name of the column in the database that holds the data for this field. If no name is set, the key will be used instead.
+   * Be aware that if you want to keep your casing, you have to escape your string with double quotes.
```
Comment (noam-honig) [inline projects/core/src/column-interfaces.ts:57]: I think this is relevant only for postgres

File: projects/core/postgres/schema-builder.ts [hunk 2]
```diff
@@ -177,13 +210,6 @@ export class PostgresSchemaBuilder {
...
-  constructor(
-    private pool: SqlDatabase,
-    schema?: string,
-  ) {
-    if (schema) {
-      this.additionalWhere = " and table_schema='" + schema + "'"
-    }
-  }
+
+  constructor(private pool: SqlDatabase) {}
```
Comment (noam-honig) [inline projects/core/postgres/schema-builder.ts:214]: You removed toe reference to schema - it's problematic - I have applications that use other schemas then public, and we need the ensure schema to work on the other schema - and it's defined on the connection level

Comment (noam-honig) [reply projects/core/postgres/schema-builder.ts:214]: See:
https://github.com/noam-honig/cfp-list/blob/master/src/server/postgres-with-schema.ts#L41

**Review by noam-honig — CHANGES_REQUESTED** [review]

File: projects/core/postgres/schema-builder.ts (hunk 2 above)
Comment (jycouet) [reply projects/core/postgres/schema-builder.ts:214]: I must not have looked well, I was thinking that it was not used.
I'll look at it 👍

File: projects/core/src/column-interfaces.ts (hunk 1 above)
Comment (jycouet) [reply projects/core/src/column-interfaces.ts:57]: Could add a note like "FYI for postgres ..."? Or you prefer to remove it? 
I think that the example is quite helpful (of course I'm on postgres ^^)

**Pull request merged**